process.c

```diff
@@ -5588,54 +5596,54 @@ proc_getrlimit(VALUE obj, VALUE resource)
 #if defined(HAVE_SETRLIMIT) && defined(NUM2RLIM)
 /*
  *  call-seq:
- *     Process.setrlimit(resource, cur_limit, max_limit)        -> nil
- *     Process.setrlimit(resource, cur_limit)                   -> nil
- *
- *  Sets the resource limit of the process.
- *  _cur_limit_ means current (soft) limit and
- *  _max_limit_ means maximum (hard) limit.
- *
- *  If _max_limit_ is not given, _cur_limit_ is used.
- *
- *  _resource_ indicates the kind of resource to limit.
- *  It should be a symbol such as <code>:CORE</code>,
- *  a string such as <code>"CORE"</code> or
- *  a constant such as Process::RLIMIT_CORE.
- *  The available resources are OS dependent.
- *  Ruby may support following resources.
- *
- *  [AS] total available memory (bytes) (SUSv3, NetBSD, FreeBSD, OpenBSD but 4.4BSD-Lite)
- *  [CORE] core size (bytes) (SUSv3)
- *  [CPU] CPU time (seconds) (SUSv3)
- *  [DATA] data segment (bytes) (SUSv3)
- *  [FSIZE] file size (bytes) (SUSv3)
- *  [MEMLOCK] total size for mlock(2) (bytes) (4.4BSD, GNU/Linux)
- *  [MSGQUEUE] allocation for POSIX message queues (bytes) (GNU/Linux)
- *  [NICE] ceiling on process's nice(2) value (number) (GNU/Linux)
- *  [NOFILE] file descriptors (number) (SUSv3)
- *  [NPROC] number of processes for the user (number) (4.4BSD, GNU/Linux)
- *  [NPTS] number of pseudo terminals (number) (FreeBSD)
- *  [RSS] resident memory size (bytes) (4.2BSD, GNU/Linux)
- *  [RTPRIO] ceiling on the process's real-time priority (number) (GNU/Linux)
- *  [RTTIME] CPU time for real-time process (us) (GNU/Linux)
- *  [SBSIZE] all socket buffers (bytes) (NetBSD, FreeBSD)
- *  [SIGPENDING] number of queued signals allowed (signals) (GNU/Linux)
- *  [STACK] stack size (bytes) (SUSv3)
- *
- *  _cur_limit_ and _max_limit_ may be
- *  <code>:INFINITY</code>, <code>"INFINITY"</code> or
- *  Process::RLIM_INFINITY,
- *  which means that the resource is not limited.
- *  They may be Process::RLIM_SAVED_MAX,
- *  Process::RLIM_SAVED_CUR and
- *  corresponding symbols and strings too.
- *  See system setrlimit(2) manual for details.
- *
- *  The following example raises the soft limit of core size to
- *  the hard limit to try to make core dump possible.
+ *    Process.setrlimit(resource, cur_limit, max_limit = cur_limit) -> nil
+ *
+ *  Sets limits for the current process for the given +resource+
+ *  to +cur_limit+ (soft limit) and +max_limit+ (hard limit);
+ *  returns +nil+.
+ *
+ *  Argument +resource+ specifies the resource whose limits are to be set;
+ *  the argument may be given as a symbol, as a string, or as a constant
+ *  beginning with <tt>Process::RLIMIT_</tt>
+ *  (e.g., +:CORE+, <tt>'CORE'</tt>, or <tt>Process::RLIMIT_CORE</tt>.
+ *
+ *  The resources available and supported are system-dependent,
+ *  and may include (here expressed as symbols):
+ *
+ *  - +:AS+: Total available memory (bytes) (SUSv3, NetBSD, FreeBSD, OpenBSD but 4.4BSD-Lite).
```

Comment:
I think the `but` here actually means `except`.

```suggestion
 *  - +:AS+: Total available memory (bytes) (SUSv3, NetBSD, FreeBSD, OpenBSD except 4.4BSD-Lite).
```